src/components/my-career/AddJobRole.jsx

```diff
@@ -50,20 +69,23 @@ const AddJobRole = () => {
               <h2>Visualize your career.</h2>
               <div className="row job-role-details">
                 <div className="col-lg-6 col-sm-12">
-                  <p>
-                    Take one minute to pick a job title that best describes your
-                    current or desired role. We&apos;ll tell you what skills you
-                    should be looking for when enrolling in courses, and track
-                    your skill growth as you complete courses.
-                  </p>
-                  <Hyperlink destination={`/${slug}/skills-quiz`}>
-                    <Icon
-                      id="add-job-role-icon"
-                      className="fa fa-plus add-job-icon"
-                      screenReaderText="Add Role"
-                    />
-                    Add Role
-                  </Hyperlink>
+                  <TransitionReplace className="mb-3">
+                    {!isEditable ? (
+                      <div key="add-job-button">
+                        <p>
+                          Take one minute to pick a job title that best describes your
+                          current or desired role. We&apos;ll tell you what skills you
+                          should be looking for when enrolling in courses, and track
+                          your skill growth as you complete courses.
+                        </p>
+                        <Button variant="link" iconBefore={addIcon} onClick={addRoleClickHandler}>Add Role</Button>
+                      </div>
+                    ) : (
+                      <div key="add-job-dropdown">
+                        <SearchJobRole onSave={onSave} onCancel={onCancel} />
```

Comment:
Can we please use more descriptive names for function like `onSaveRole`

Reply:
Done